### PR TITLE
Fix hostports tests to work with hyperbahn

### PR DESCRIPTION
**Pull request by Raynos**
The hyperbahn cluster is slightly different. This gets the
tests to pass from Hyperbahn itself

r: @jcorbin @rf @shannili

### CODE REVIEW

File: node/test/hyperbahn/hostports.js [hunk 1]
```diff
@@ -207,6 +211,7 @@ function runTests(HyperbahnCluster) {
                 'error message should be a parsing failure');
 
             var items = cluster.logger.items();
+            console.log('items', items);
```
Comment (kriskowal) [inline node/test/hyperbahn/hostports.js]: console

**Comment by kriskowal** [comment]
This screams `getOrCreateSubChannel` or simply `getChannel` with the rest implied. cc @jcorbin

**Comment by jcorbin** [comment]
yes please, let's add a 'getOrMake' (or s/make/create/ to taste).

**Comment by ShanniLi** [comment]
+1 for getOrMake. However, it feels optional for this diff.

lgtm.

**Comment by Raynos** [comment]
Disagree with getOrMake.

This is just a temporary mess because two versions of TestCluster.

**Pull request merged**